std/experimental/testing/io.d

```diff
@@ -0,0 +1,312 @@
+/**
+ * IO related functions
+ */
+
+module std.experimental.testing.io;
```

Comment:
Yes, possibly. I wrote all of this long before `std.experimental.logger` existed, of course. I'm going to have to look into it.
It might just be that I need another logger backend for multithreaded applications first, though.